src/index.html

```diff
@@ -234,30 +234,60 @@ <h2>SKILLS</h2>
       </section>
 
       <!-- Information -->
-      <div>
-        <section>
-          <h2>QUICK ID</h2>
-          <ul>
-            <li><b>Phone</b><a href="tel:123-456-7890">123-456-7890</a></li>
-            <li>
-              <b>Email</b><a href="mailto:info@mysite.com">info@mysite.com</a>
-            </li>
-            <li>
-              <b>Website</b
-              ><a href="http://www.wix.com" target="_blank">www.wix.com</a>
-            </li>
-            <li>
-              <b>Address</b>
-              <address>
-                500 Terry Francois Street San Francisco, CA 94158
-              </address>
-            </li>
-          </ul>
-        </section>
+      <div id="banner_information">
```

Comment:
You just need one ID `banner_information` here, if you have IDs everywhere then it is against the purpose.
Have one ID and select its child is better.

But personally I still like to extract all common styles and assign them into css class to reuse.